examples/demo-app/src/utils/url.js

```diff
@@ -39,3 +39,45 @@ export function parseQueryString(query) {
 export function getMapPermalink(mapLink) {
   return `${window.location.protocol}//${window.location.host}/${MAP_URI}${mapLink}`
 }
+
+// from http://blog.stevenlevithan.com/archives/parseuri
+/**
+ * Allows to break down a url into multiple params
+ * @param str
+ */
+export function parseUri (str) {
```

Comment:
There is a nice little library that does this
https://www.npmjs.com/package/query-string

Reply:
I look into the library, unfortunately it can only parse the query param part of URI.

I wanted something like: 
parse(http://www.whatever.com/index.csv?param1=1&param2=2 and return something)

```
{
  host: www.whatever.com,
  file: index.csv,
  params: {
    param1=1,
    param2=2
  }
}
```

Reply:
I don't get it, isn't your `mapUrl=file.csv` also part of the query param?